### PR TITLE
Added FormViewSelector

### PR DESCRIPTION
**Pull request by bmartins-42**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

Added a `FormViewSelector` that is taking `v-model`, `option1`, `option2` as required and `option3` as optional

image image

### Related issue

- [#ISSUE_NUMBER](https://github.com/activist-org/activist/issues/456)

### CODE REVIEW

**Comment by bmartins-42** [comment]
- I do not have a lot of experience with A11Y so let me know if the compatibility has to be reworked.
- I also not sure about the color I used so let me no if I need to change them

**Comment by andrewtavis** [comment]
Thanks for this, @bmartins-42! I'll try to get back to you with a review soon :)

File: frontend/i18n/en-US.json [hunk 1]
```diff
@@ -342,8 +345,12 @@
       "contains-numbers": "Contains numbers",
       "contains-special-chars": "Contains special characters (!#$^*)"
     },
-    "marker-private": {
-      "private": "Private"
+    "view-selector": {
+      "title-aria-label": "Select a view type",
```
Comment (andrewtavis) [inline frontend/i18n/en-US.json:349]: This is really really wonderful, @bmartins-42 😊 Thanks for the focus on accessibility!

**Review by andrewtavis — APPROVED** [review]
Really great work here, @bmartins-42 😊 Just tried it out and it's perfect. Hope that the conversion over the Headless UI wasn't too annoying! Would be great to get your feedback on Headless. If you enjoyed using it, then there are a few more conversion issues that were just made, and it'd also be great to see if there are other components that we should be using it in!

**Pull request merged**